src/python/pants/backend/jvm/targets/jar_library.py

```diff
@@ -77,7 +77,7 @@ def excludes(self):
     return self.payload.excludes
 
   @property
-  def exports(self):
+  def exports_targets(self):
```

Comment:
hmm, this wasn't added very long ago (about a month ago), I don't think anyone is using it yet, but I can also keep it and make it "return None" I guess?